charts/accurate/templates/rolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "accurate.fullname" . }}-leader-election-rolebinding
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "accurate.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ template "accurate.fullname" . }}-leader-election-role
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "accurate.fullname" . }}-controller-manager
```

Comment:
What happens if the user chooses not to create a service account?

Reply:
refs: https://github.com/cybozu-go/accurate/pull/11#discussion_r703965662